src/components/PercentageLine/Percentageline.test.jsx

```diff
@@ -0,0 +1,12 @@
+import { describe, it, expect } from "vitest";
+import { render, screen } from "@testing-library/react";
+import PercentageLine from "./PercentageLine";
+
+describe("<PercentageLine />", () => {
+  it("should render the progress bar", () => {
+    render(<PercentageLine percentage={50} />);
+
+    expect(screen.getByRole("progressbar")).toBeInTheDocument();
```

Comment:
You don't need to check presence here, since you're already checking details bellow